types/xo/package.json

```diff
@@ -6,7 +6,7 @@
         "https://github.com/xojs/xo#readme"
     ],
     "dependencies": {
-        "@types/eslint": "*"
+        "@types/eslint": "8"
```

Comment:
Intentional change?

Reply:
Yes, I had to pin the dependency to `"@types/eslint": "8"` because after my changes some files in `types/xo` were failing the CI build. This can be seen in [this job run](https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/9749705290/job/26907506292).